### PR TITLE
Fix cms

### PR DESCRIPTION
**Pull request by n8225**
Thank you for taking the time to work on a PR for Awesome-selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

Issues #35 #471

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,20 +350,20 @@ _[Extensible Messaging and Presence Protocol](https://en.wikipedia.org/wiki/XMPP
 
 CMS are a practical way to setup a website with many features. CMS often come with third party plugins, themes and functionality that is easy to add and customize to your needs. See also [Blogging Platforms](#blogging-platforms) and [Static Site Generators](#static-site-generators)
 
-  * [99ko CMS](http://www.99ko.org/) - 99ko allows you to create a website in minutes without knowledge of Web development. It does not use a database server but relies on simple text files (JSON format) to store your data. ([Source Code](https://github.com/99ko-project/99ko-cms))
-  * [Apostrophe 2](http://apostrophenow.org/) - A node.js CMS with a focus on extensible in-context editing tools. ([Demo](http://demo.apostrophenow.com/), [Source Code](https://github.com/punkave/apostrophe)) `MIT` `NodeJS`
+  * [99ko CMS](http://www.99ko.org/) - 99ko allows you to create a website in minutes without knowledge of Web development. It does not use a database server but relies on simple text files (JSON format) to store your data. ([Source Code](https://github.com/j84/99ko)) `NoLicense` `PHP`
```
Comment (nodiscc) [inline README.md]: I've reported https://github.com/j84/99ko/issues/5 to attempt to fix the missing license. If the maintainers respond quickly enough could you update the license here? If it takes too long I suggest removing 99ko (at least temporarily)

**Comment by nodiscc** [comment]
Great work, thanks. There's just the license problem mentioned above, let's wait?

**Comment by n8225** [comment]
Due to no response from https://github.com/j84/99ko/issues/5 I deleted 99ko.  I will also create an issue to add 99ko for when/if a license is added.

**Comment by Kickball** [comment]
Looks good, thanks.

**Pull request merged**